plaidml2/bridge/keras/__init__.py

```diff
@@ -1547,8 +1557,21 @@ def square(x):
 
 
 @_log_call
-def squeeze(x, axis):
-    return _KerasNode('squeeze', tensor=plaidml_op.squeeze(x.tensor, axis))
+def squeeze(x, axis=None):
+    if axis is None:
+        # define axis
+        axis = []
+        for s in range(len(x._keras_shape)):
+            if x._keras_shape[s] == 1:
```

Comment:
Replace with `int_shape(x)[s]` here. Since we're directly comparing to the python integer 1 this won't catch symbolic shapes that are 1 anyway, so just use `int_shape` to be clear about that.